cves/CVE-2014-0098.yml

```diff
@@ -66,15 +70,18 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 4bab699bdccdd3f48943d6ae224a1253a9a1a0d2
+   - commit: 0559370e7e2bf322d11d6df51b934d211fb2465c
+     note: SVN rev 1575400, from the Apache website.
+   - commit:
      note:
-   - commit: 70262b7452d6f5b6f8e38dffff5d2e95c9ae2456
-     note:
-   - commit: 83f5a29060001bd407e243b4a19181bc0fe1b3d4
+   - commit:
      note:
 vccs:
-  - commit:
-    note:
+  - commit: 066712a8cdcb414467706def84e461b393af503c
+    note: |
+      This commit was the most recent one before Apache 2.4.1 was released,
```

Comment:
The wording here is a little odd. Lot's of "was" and the trailing SVN rev sitting by itself at the end. Maybe a refactor of the couple of sentences here could increase readability a bit?